Plugins/Flow.Launcher.Plugin.Sys/Main.cs

```diff
@@ -192,6 +192,156 @@ private List<Result> Commands()
                     }
                 },
                 new Result
+                {
+                    Title = "Explorer",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_explorer"),
+                    IcoPath = "Images\\explorer.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xec50"),
+                    Action = c =>
+                    {
+                        {
+                            System.Diagnostics.Process.Start("explorer.exe", "");
+                        }
+
+                        return true;
+                    }
```

Comment:
is this one needed? Isn't the program plugin already indexed the explorer?

Reply:
We don't need it. removed.